README.md

```diff
@@ -49,10 +47,9 @@ Three types of data are associated with the dataset: clique metadata (*Discogs-V
 
 ### Metadata
 
-TODO zip the files, describe the contents in a readme file inside the main directory. add a license.
 TODO upload to zenodo, add the url here.
 
-We provide all the metadata including the intermediary files of the dataset creation process and the final outputs. Due to their sizes they are separated into two directories so that one does not have to download everything. If your goal is to download the main metadata and start working, download `discogs_20240701/main/` (21GB before compressing). If for some reason you are interested in the intermediary files, you download `discogs_20240701/intermediary/` (46GB before compressing). Contents of these folders are provided in [this section](#data-structure).
+We provide the dataset including the intermediary files of the creation process. Due to their sizes they are separated into two directories so that an intereseted person do not have to download everything. If your goal is to download the main metadata and start working, download `discogs_20240701/main/` (21GB before compressing). If for some reason you are interested in the intermediary files, you download `discogs_20240701/intermediary/` (46GB before compressing). Contents of these folders are provided in [this section](#data-structure).
```

Comment:
We could simply remove " so that an intereseted person do not have to download everything" part of the phrase (simpler).

Reply:
Simpler:
"46 GB before compressing" -> "46 GB uncompressed"
"21 GB before compressing" -> "21 GB uncompressed"

Reply:
Simpler: "We provide the dataset including" -- >"The dataset includes intermediary files of the creation process."